formapp/app.py

```diff
@@ -1232,12 +1238,12 @@ def endpoint_help():
         name = sD.studentDict[request.remote_addr]['name']
         name = name.strip()
         if name in helpList:
-            return redirect("/chat?alert=You already have a help ticket or break request in. If your problem is time-sensitive, or your last ticket was not cleared, please get the teacher's attention manually." )
+            return redirect("/help?alert=You already have a help ticket or break request in. If your problem is time-sensitive, or your last ticket was not cleared, please get the teacher's attention manually." )
```

Comment:
Will this be able to handle spaces in the URL. This is why I used templates. Like EJS, they inject the text into the HTML before sending, rather than sending extra data to be parsed.

Reply:
Yes, the URLs work with the spaces. Showing an alert and redirecting to another page should be simpler than having a separate message page.